common.yml

```diff
@@ -88,14 +88,17 @@
   post_tasks:
     - name: Register node.
       shell: >
-        tsuru-admin docker-node-add --register address=http://{{ gce_private_ip }}:{{ docker_port }}
+        tsuru-admin docker-node-add --register address=http://{{ gce_private_ip }}:{{ docker_port }} ;
+        tsuru-admin docker-node-update http://{{ gce_private_ip }}:{{ docker_port }} pool=default
```

Comment:
Could you break these out to separate tasks?

We've had problems with multiple shell commands in single tasks. Ansible only reports the exit code of the last command, so it makes it quite hard to tell if something has gone wrong and what it was.